src/blocking/runnable.rs

```diff
@@ -0,0 +1,28 @@
+use crate::queue::Queueable;
+use crate::Error;
+use chrono::DateTime;
+use chrono::Utc;
+
+pub const COMMON_TYPE: &str = "common";
+
+pub enum Scheduled {
```

Comment:
I think it can be lib.rs because it's shared between blocking and asynk

Reply:
[Finishing blocking module, starting to modify tests](https://github.com/ayrat555/fang/pull/74/commits/7d1402f52f94e1d0520746e15693116b5988ed4f)